mscore/albummanager.cpp

```diff
@@ -599,33 +799,47 @@ void AlbumManager::removeClicked(bool throwaway)
 //   deleteClicked
 //---------------------------------------------------------
 
-void AlbumManager::deleteClicked(bool throwaway)
+void AlbumManager::deleteClicked(bool checked)
 {
+    Q_UNUSED(checked);
```

Comment:
I presume there will eventually be an implementation here ;)

Reply:
:laughing: probably, but I might remove that button.